### ISSUE TITLE
New lines of code announced as buttons by screenreader

### ISSUE THREAD

**Issue by outofambit** [desktop/desktop#6577]
Screenreader announces "20, button" without a clear understanding. Other changes are not announced as buttons.

![tdna_gh_desk_win_043](https://user-images.githubusercontent.com/964912/50985634-a74f2d00-14b9-11e9-9c99-031f987a8165.png)

They are buttons but perhaps we need to turn them into checkboxes or something and add more screenreader-accessible text to explain that it's toggling the inclusion of a line

**Comment by j-f1** [comment]
“Line X. (Addition/Removal) (will/will not) be committed. Click to toggle”?

**Comment by tidy-dev** [comment]
This was fixed in the recent accessibility improvements to the diff.
